README.rst

```diff
@@ -107,6 +107,23 @@ this would simply be ``'report'``) and ``tor_datadir`` (where you would
 like the spawned Tor process to keep its data). If you compiled Tor yourself,
 you'll likely want to specify it for the ``tor_binary`` option.
 
+Configure bouncer and collector endpoints
+.........................................
+
+The bouncer and collector are HTTP applications ("protocols" in twisted terminology) that can be configured to run on top of plain TCP, TLS, or onion service endpoints.
+Here is an example of the relevant part of the configuration::
+    bouncer_endpoints:
+    - {type: tls, port: 10443, cert: "private/ssl-key-and-cert.pem"}
+    - {type: tcp, port: 10080}
+    - {type: onion, hsdir: "/some/private/bouncer"}
```

Comment:
I didn't know one could avoid indenting: I pythonically assumed one had to :-).

> Do you prefer indented?

No, no need to make changes: I think whatever is compliant with YAML and consistent with the rest of the file is OK.